assets/app/features/upgrade/controllers/upgrade-backup.controller.js

```diff
@@ -68,18 +75,9 @@ function() {
                             // Determine the content type from the header
                             contentType = headers['content-type'],
 
-                            backupBlob = new Blob([response.data], {type: contentType}),
-                            backupObjectUrl = URL.createObjectURL(backupBlob),
+                            backupBlob = new Blob([response.data], {type: contentType});
```

Comment:
Hmmm... haven't heard about such requirement. I tested `responseType: blob` in current FF and Chrome and it works correctly.